Anchal_Hora/nodejs/UI/create.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
why are you not using Bootstrap ?

Reply:
Using bootstrap in the main html file that is index.html